File: app/src/main/java/com/dvor/my/mydvor/NewsBD.java
package com.dvor.my.mydvor;

public class NewsBD {

    public String header;//authorID;
    public String text;
    public String date;
    public String img;
    public String uID;
    public NewsBD() {
        // Default constructor required for calls to DataSnapshot.getValue(User.class)
    }

    public NewsBD( String  header, String text, String date, String img,
                  String uID) {
        this. header =  header;
        this.text = text;
        this.date=date;
        this.img=img;
        this.uID=uID;
    }

}

File: app/src/main/java/com/dvor/my/mydvor/data/NewsBD.kt
package com.dvor.my.mydvor.data

import androidx.annotation.Keep
import com.google.firebase.database.IgnoreExtraProperties

@IgnoreExtraProperties
@Keep
data class NewsBD(
        var header: String?,
        var text: String?,
        var date: String?,
        var img: String?,
        var uid: String?
)